src/content/activity/Conservation.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Conservation
+image: ./dummy.svg
+---
+
+The sustainable use and management of natural resources, including wildlife, water, air, and earth deposits, to ensure their availability to present and future generations.
```

Comment:
could conservation include an area or resource that is not used or actively managed, more simply preserved or protected? if that is the case then it could be different from sustainable use and management.

Reply:
Wondering the same. @nedhorning maybe it would make more sense to define it as, 

The protection, preservation, management, or restoration of natural environments and the ecological 
 communities that inhabit them. Conservation is generally held to include the sustainable use and management of natural resources including wildlife, water, air, and earth deposits, to ensure their availability to present and future generations.

Reply:
I followed Sam's definition but removed "restoration" since that's covered in EcosystemRestoration.